### PR TITLE
No re-render of Call composite if chat/people pane is opened and closed

### PR DESCRIPTION
**Pull request by mgamis-msft**
# What

Hiding inner call composite in CallWithChat composite instead of unmounting and re-rendering if chat/people pane is opened and closed. 

# Why

Dismissed compliance banners will reappear if chat/people pane is opened and closed.
Before: 
https://user-images.githubusercontent.com/79475487/162343097-7321bce7-4859-4291-96a9-7b6822e66208.mp4
After:
https://user-images.githubusercontent.com/79475487/162343196-195d4cee-3610-4394-a1f2-aeb22d4ca9d2.mp4

# How Tested

Locally

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx [hunk 1]
```diff
@@ -242,17 +243,15 @@ const CallWithChatScreen = (props: CallWithChatScreenProps): JSX.Element => {
   return (
     <Stack verticalFill grow styles={compositeOuterContainerStyles} id={compositeParentDivId}>
       <Stack horizontal grow>
-        {!isMobileWithActivePane && (
```
Comment (JamesBurnside) [inline packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx:245]: Out of interest, if keyboard navigating, where does focus go when the side pane is closed?

Comment (JamesBurnside) [inline packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx:245]: Can you test also, in mobile, does the PiPiP still work as expected? I'm worried the video tiles on the gallery will still have handles to the streams and maybe won't load the streams of the PiPiP correctly

**Comment by prprabhu-ms** [comment]
waiting for answers to pending questions before I approve.

File: packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx (hunk 1 above)
Comment (mgamis-msft) [reply packages/react-composites/src/composites/CallWithChatComposite/CallWithChatComposite.tsx:245]: The Call local stream stops working after opening and closing the chat/people pane.
https://user-images.githubusercontent.com/79475487/164388812-4d1c3dbc-66af-4ab9-98ca-5e988ffde635.mp4
The fix will to be done to the streams in the Call without rerendering the Call. 

Closing this PR. 

Logged a bug for the Compliance Banner reappearing https://skype.visualstudio.com/SPOOL/_search?text=Dismissed*&type=workitem&lp=workitems-Project&filters=Projects%7BSPOOL%7D&pageSize=25.

**Pull request closed without merging**